packages/offix-datastore/src/Model.ts

```diff
@@ -17,6 +18,40 @@ export type Fields<T> = {
     [P in keyof T]: FieldOptions
 };
 
+/**
+ * Model Config options
+ */
+export interface ModelConfig<T = unknown> {
+    /**
+     * Model name
+     */
+    name: string;
+
+    /**
+     * Model store name, defualts to `user_${name}`
```

Comment:
`data_${name}` seems to make more sense (and be more innovative)

Reply:
Really anything you like